ui_framework/doc_site/src/views/tool_bar/tool_bar_example.jsx

```diff
@@ -1,26 +1,35 @@
-import React, {
-  Component,
-  PropTypes,
-} from 'react';
+import React from 'react';
+
+import { renderToHtml } from '../../services';
 
 import {
   GuideDemo,
-  GuideLink,
   GuidePage,
   GuideSection,
   GuideSectionTypes,
   GuideText,
 } from '../../components';
 
-const toolBarHtml = require('./tool_bar.html');
-const toolBarSearchOnlyHtml = require('./tool_bar_search_only.html');
-const toolBarFooterHtml = require('./tool_bar_footer.html');
+import { ToolBar } from './tool_bar';
+const toolBarSource = require('!!raw!./tool_bar');
+const toolBarHtml = renderToHtml(ToolBar);
+
+import { ToolBarSearchOnly } from './tool_bar_search_only';
+const toolBarSearchOnlySource = require('!!raw!./tool_bar_search_only');
+const toolBarSearchOnlyHtml = renderToHtml(ToolBarSearchOnly);
+
+import { ToolBarFooter } from './tool_bar_footer';
+const toolBarFooterSource = require('!!raw!./tool_bar_footer');
+const toolBarFooterHtml = renderToHtml(ToolBarFooter);
 
 export default props => (
   <GuidePage title={props.route.name}>
     <GuideSection
       title="ToolBar"
       source={[{
+        type: GuideSectionTypes.JS,
+        code: toolBarSource,
+      }, {
```

Comment:
This is not a comment on _this_ PR, but just an idea I had for `GuideSection` itself (cc @cjcenizal). Maybe it could look something like this instead:

```js
<GuideSection title="ToolBar">
  <GuideSectionSource
    type={GuideSectionTypes.JS}
    code={toolBarSource} />
  <GuideSectionSource
    type={GuideSectionTypes.HTML}
    code={toolBarHtml} />
</GuideSection>
``` 

or even better, taking it as far as:

```js
<GuideSection title="ToolBar">
  <JavaScript code={toolBarSource} />
  <Html code={toolBarHtml} />
</GuideSection>
```

So clean and nice 🚀 🌔